src/structures/GuildChannel.js

```diff
@@ -414,8 +406,7 @@ class GuildChannel extends Channel {
   setParent(channel, { lockPermissions = true, reason } = {}) {
     return this.edit(
       {
-        // eslint-disable-next-line no-prototype-builtins
-        parentID: channel !== null ? (channel.hasOwnProperty('id') ? channel.id : channel) : null,
+        parentID: channel !== null ? ('id' in channel ? channel.id : channel) : null,
```

Comment:
This is the only time I recall seeing us doing `hasOwnProperty` / `in` for this.. Don't we usually `channel.id ?? channel`?